language/move-binary-format/src/file_format.rs

```diff
@@ -62,6 +62,7 @@ macro_rules! define_index {
         #[derive(Clone, Copy, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
         #[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
         #[cfg_attr(any(test, feature = "fuzzing"), proptest(no_params))]
+        #[cfg_attr(any(test, feature = "arbfuzz"), derive(arbitrary::Arbitrary))]
```

Comment:
Can we roll up to line 63?

Reply:
scratch that, I'm dumb

Reply:
You are right this reads pretty bad, so I made it `derive(proptest_derive::Arbitrary)` for differentiation. Also now under the same `fuzzing` feature which does not hurt.